src/features/Faucet/types.ts

```diff
@@ -0,0 +1,26 @@
+import { ITxData, ITxGasLimit, ITxGasPrice, ITxHash, TAddress } from '@types';
```

Comment:
can this go to the `types` folder ?

Reply:
Why? Isn't it used exclusively for this feature?

Reply:
I assumed we used some of those outside, if not then it can live here